WordPress/src/main/java/org/wordpress/android/viewmodel/pages/PageListViewModel.kt

```diff
@@ -154,7 +154,7 @@ class PageListViewModel @Inject constructor(
 
     fun onMenuAction(action: Action, pageItem: Page, context: Context): Boolean {
         completeEditHomePageTour(pageItem, context)
-        return pagesViewModel.onMenuAction(action, pageItem)
+        return pagesViewModel.onMenuAction(action, pageItem, context)
```

Comment:
@EmilyGoose

No changes requested, but I did want to call out the general use of `context`.

It is preferable not to pollute a `viewModel` with `context` (or any android related frameworks); viewmodels should remain agnostic. This aligns more closely with Clean Code and has the added benefit of making the VM more easily testable. If `context` was not already present in `PageListViewModel`, I would have suggested a solution along the following lines:  Have the fragment observe a livedata and perform the copy action itself in the fragment when the event is raised. 

Thanks.

Reply:
I did consider pulling out the functionality to an observer pattern but ended up settling on passing context to the function in order to touch the least source files with this change.